translations/es/LC_MESSAGES/messages.po

```diff
@@ -1121,7 +1119,7 @@ msgstr "Lo último"
 
 #: /content/includes/latest.yaml /content/latest/latest.html
 msgid "The Latest News"
-msgstr "Las Últimas Noticias"
+msgstr "Últimas Noticias"
 
```

Comment:
@bpaduch, here maybe you should change the english too? Instead "The latest news", be "Latest News" ??

Reply:
That English string is pulled in from the page title (the html `<title>` tag) - https://www.ampproject.org/latest/. I suggest we leave it, otherwise we'll have to change all other translation files.

Reply:
Ok :heart: